components/bl-mention-component/src/index.js

```diff
@@ -0,0 +1,99 @@
+import { useCallback, useEffect, useState } from 'react';
+
+import primereact from './lib/core';
+import { filterSuggestions, orderFields, stringToList, useTriggers } from './helpers.js';
+import { SuggestionCard } from './suggestion-card.js';
+
+const { Mention } = primereact.mention;
+const { cn } = BackendlessUI.CSSUtils;
+
+const BLACKLISTED_FIELDS = new Set(['created', '___class', 'ownerId', 'updated', 'objectId']);
+
+export default function MentionComponent({ component, eventHandlers }) {
+  const {
+    trigger, suggestions, field, scrollHeight, autoHighlight, placeholder, delay, autoresize, rows, cols,
```

Comment:
`component.suggestions` means that we have have loaded all suggestions on the client? what if there are +100500 users?